docs/style.css

```diff
@@ -275,6 +272,16 @@ li {
 	border-bottom: 0;
 }
 
+#cgit > .content {
+	margin: 1rem 0;
+}
+
+#cgit > .content > .commit-info,
+#cgit > .content > .commit-msg,
```

Comment:
<img width="1209" alt="Screenshot 2024-01-25 at 23 24 19" src="https://github.com/underground-software/orbit/assets/1909841/8561e0c6-47fb-4cac-b317-559c899332af">

A bit funky when commit msg is single line. Maybe hide it? Open to suggestions